alphadia/constants/default.yaml

```diff
@@ -158,6 +158,29 @@ scoring_config:
   precursor_mz_tolerance: 10
   fragment_mz_tolerance: 15
 
+# perform non-isobaric multiplexing of any input library
+library_multiplexing:
+  # if true, the library is multiplexed
+  enabled: False
+
+  # define channels by their name and how modifications should be translated from the input library to the multiplexed library
+  multiplex_mapping:
+    0:
```

Comment:
(nit) these ints are basically the channel names, i.e. could this in theory also be an arbitrary string? Maybe a small comment on this, as I was initially confused by the ints